enos/modules/vault_verify_write_data/scripts/smoke-enable-secrets-kv.sh

```diff
@@ -30,8 +30,19 @@ binpath=${vault_install_dir}/vault
 
 test -x "$binpath" || fail "unable to locate vault binary at $binpath"
 
-export VAULT_ADDR='http://127.0.0.1:8200'
-export VAULT_TOKEN='${vault_token}'
-
 retry 5 "$binpath" status > /dev/null 2>&1
-retry 5 $binpath secrets enable -path="secret" kv
+
+# Create user policy
+$binpath policy write reguser -<<EOF
+path "*" {
+  capabilities = ["read", "list"]
+}
+EOF
+
+# Enable the userpass auth method
```

Comment:
I don't have a problem with adding a regular user, I'm just curious why we do it for testing the k/v secrets engine.

Reply:
That's a great question, I'll add a comment clarifying this.  So we add a regular user here because we also use this module to verify data in replication scenario.  When replication is enabled, we cannot authenticate to the secondary server using the root token, so I add a user here so we can authenticate and verify written data for all scenarios

Reply:
added comments